docs/rules/replace-j-unit-assert-that-with-hamcrest.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Replace JUnit assertThat with Hamcrest
+ruleId: ReplaceJUnitAssertThatWithHamcrest
+since: 3.29.0
+minJavaVersion: 5
+remediationCost: 5
```

Comment:
I will correct this value to '2' in order to be consistent with the Java implementation.